src/components/item/iterable.js

```diff
@@ -177,12 +209,19 @@ class ItemIterable extends React.PureComponent {
     getSelection: func.isRequired,
 
     onContextMenu: func.isRequired,
+    onDragStart: func.isRequired,
+    onDragStop: func.isRequired,
     onDropItems: func.isRequired,
     onDropPhotos: func.isRequired,
     onItemOpen: func.isRequired,
     onPhotoError: func.isRequired,
     onSelect: func.isRequired
   }
+
+  static defaultProps = {
```

Comment:
Since you added the extra checks before invoking the two callbacks, we don't need to make them required and set defaults. I normally add the `noop` defaults only to avoid the extra checks, before invoking callbacks.

Reply:
Please remove the `noop` defauts and mark the two functions as not required.